components/openweather_api/actions/get-weather-forecast-by-location/get-weather-forecast-by-location.mjs

```diff
@@ -0,0 +1,105 @@
+import { ConfigurationError } from "@pipedream/platform";
+import openweatherApi from "../../openweather_api.app.mjs";
+
+export default {
+  key: "openweather_api-get-weather-forecast-by-location",
+  name: "Get current weather forcast by location",
+  description: `Retrieves 1-16 days weather forecast for a specified location. 
+  [See the docs here](https://openweathermap.org/forecast16#geo16). For more accurate reading, you are adviced to fill in the country code and/orstate code `,
+  version: "0.0.1",
+  type: "action",
+  props: {
+    openweatherApi,
+    appid: {
+      propDefinition: [
+        openweatherApi,
+        "appId",
+      ],
+    },
```

Comment:
Since we can get the api key from `this.openweatherApi.$auth.api_key`, this prop isn't needed.
```suggestion
    },
```